host/src/CMakeLists.txt

```diff
@@ -15,18 +15,14 @@ set(CMAKE_TOOLCHAIN_FILE "${vcpkg_SOURCE_DIR}/scripts/buildsystems/vcpkg.cmake"
 
 set(VCPKG_INSTALL_OPTIONS "--debug")
 
-string(TIMESTAMP CURRENT_TIME "%m%d%H%M")
```

Comment:
Do not need this approach anymore as native host code will be released as versioned nuget packages.